src/plugins/custom_import_map/public/components/empty_prompt.test.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import EmptyPrompt from './empty_prompt';
+
+describe('empty_prompt', () => {
+  it('renders the Empty Prompt based on props provided', () => {
+    const wrapper = shallow(
+      <EmptyPrompt iconType="test" title="test" bodyFragment="test" actions="" />
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
Not for this PR again, but avoid using snapshot tests unless you really want to. It can be more painful than useful in the long run. Snapshots are really useful if the whole snapshot is readable and not meant to change. Components and styles are not good use cases for them. Config values and schema values are better suited for snapshots.

Reply:
Thank you @ashwin-pc, it makes a lot more sense now. I'll keep this in for now just to have some test running, in the subsequent PRs, this component will be removed and so the test too : )